src/System.Diagnostics.Process/tests/ProcessTests.cs

```diff
@@ -1453,7 +1529,11 @@ public void Process_StartTest()
             Assert.Equal(userName, p.StartInfo.UserName);
             Assert.Same(password, p.StartInfo.Password);
             Assert.Equal(domain, p.StartInfo.Domain);
+
+            // Make sure that process is fully created before killing it to avoid AccessDeniedExceptions.
+            Thread.Sleep(500);
```

Comment:
```
 Discovering: System.Diagnostics.Process.Tests
  Discovered:  System.Diagnostics.Process.Tests
  Starting:    System.Diagnostics.Process.Tests
     System.Diagnostics.Tests.ProcessTests.TestInvalidPriorityClass(priorityClass: 0) [FAIL]
        Assert.Throws() Failure
        Expected: typeof(System.ComponentModel.InvalidEnumArgumentException)
        Actual:   typeof(System.ArgumentException): The value of argument 'value' (0) is invalid for Enum type 'System.
  Diagnostics.ProcessPriorityClass'.
        Stack Trace:
           C:\Users\vikto\Documents\src\corefx\src\System.Diagnostics.Process\src\System\Diagnostics\Process.cs(485,0):
   at System.Diagnostics.Process.set_PriorityClass(ProcessPriorityClass value)
           C:\Users\vikto\Documents\src\corefx\src\System.Diagnostics.Process\tests\ProcessTests.cs(711,0): at System.D
  iagnostics.Tests.ProcessTests.<>c__DisplayClass53_0.<TestInvalidPriorityClass>b__0()
     System.Diagnostics.Tests.ProcessTests.TestInvalidPriorityClass(priorityClass: 96) [FAIL]
        Assert.Throws() Failure
        Expected: typeof(System.ComponentModel.InvalidEnumArgumentException)
        Actual:   typeof(System.ArgumentException): The value of argument 'value' (96) is invalid for Enum type 'System
  .Diagnostics.ProcessPriorityClass'.
        Stack Trace:
           C:\Users\vikto\Documents\src\corefx\src\System.Diagnostics.Process\src\System\Diagnostics\Process.cs(485,0):
   at System.Diagnostics.Process.set_PriorityClass(ProcessPriorityClass value)
           C:\Users\vikto\Documents\src\corefx\src\System.Diagnostics.Process\tests\ProcessTests.cs(711,0): at System.D
  iagnostics.Tests.ProcessTests.<>c__DisplayClass53_0.<TestInvalidPriorityClass>b__0()
  Finished:    System.Diagnostics.Process.Tests
```

I have absolutely no idea... Where was it fixed? In coreclr or corefx or some other dependencies?